public/app/features/migrate-to-cloud/onprem/ResourceDetailsModal.tsx

```diff
@@ -4,49 +4,56 @@ import { Trans, t } from 'app/core/internationalization';
 import { prettyTypeName } from './TypeCell';
 import { ResourceTableItem } from './types';
 
-interface ResourceErrorModalProps {
+interface ResourceDetailsModalProps {
   resource: ResourceTableItem | undefined;
   onClose: () => void;
 }
 
-export function ResourceErrorModal(props: ResourceErrorModalProps) {
+export function ResourceDetailsModal(props: ResourceDetailsModalProps) {
   const { resource, onClose } = props;
 
   const refId = resource?.refId;
   const typeName = resource && prettyTypeName(resource.type);
 
+  let msgTitle = 'Resource migration details:';
+  if (resource?.status === 'ERROR') {
+    msgTitle = 'Unable to migrate this resource:';
+  } else if (resource?.status === 'WARNING') {
+    msgTitle = 'Resource migrated with a warning:';
+  }
+
   return (
     <Modal
-      title={t('migrate-to-cloud.resource-error.title', 'Unable to migrate this resource')}
+      title={t('migrate-to-cloud.resource-details.title', 'Migration resource details')}
       isOpen={Boolean(resource)}
       onDismiss={onClose}
     >
       {resource && (
         <Stack direction="column" gap={2} alignItems="flex-start">
           <Text element="p" weight="bold">
-            <Trans i18nKey="migrate-to-cloud.resource-error.resource-summary">
+            <Trans i18nKey="migrate-to-cloud.resource-details.resource-summary">
               {{ refId }} ({{ typeName }})
             </Trans>
           </Text>
 
-          {resource.error ? (
+          {resource.message ? (
             <>
               <Text element="p">
-                <Trans i18nKey="migrate-to-cloud.resource-error.specific-error">The specific error was:</Trans>
+                <Trans i18nKey="migrate-to-cloud.resource-details.specific-message-title">{msgTitle}</Trans>
```

Comment:
We want a translated string per message.

Instead of 

```jsx
let msgTitle = 'Resource migration details:';
if (resource?.status === 'ERROR') {
  msgTitle = 'Unable to migrate this resource:';
} else if (resource?.status === 'WARNING') {
  msgTitle = 'Resource migrated with a warning:';
}

...

<Text>
  <Trans i18nKey="...">{msgTitle}</Trans>
</Text>
```

we should instead do

```jsx
let msgTitle = t('migrate-to-cloud.resource-details.generic-title', 'Resource migration details:');
if (resource?.status === 'ERROR') {
  msgTitle = t('migrate-to-cloud.resource-details.error-title', 'Unable to migrate this resource:');
} else if (resource?.status === 'WARNING') {
  msgTitle = t('migrate-to-cloud.resource-details.warning-title', 'Resource migrated with a warning:');
}
...

<Text>
  {msgTitle}
</Text>
```

Reply:
and run `make i18n-extract` after 😌

Reply:
@joshhunt thanks for the guidance! Changed it here and in `public/app/features/migrate-to-cloud/onprem/Page.tsx` [a16e38e](https://github.com/grafana/grafana/pull/91545/commits/a16e38e7f34a6b044cb3d1ab910f7ffecd6b98fa)